pkg/objectcache/containerprofilecache/containerprofilecache.go

```
@@ -116,6 +114,14 @@ type ContainerProfileCacheImpl struct {
 	// deprecationDedup tracks (kind|ns/name@rv) keys to emit one WARN log
 	// per legacy CRD resource-version across the process lifetime.
 	deprecationDedup sync.Map
+
+	// Projection spec — installed by SetProjectionSpec when rulemanager loads rules.
+	currentSpecMu         sync.RWMutex
+	currentSpec           *objectcache.RuleProjectionSpec
+	specGeneration        atomic.Int64  // bumped on each distinct spec hash change
+	convergenceCount      atomic.Int64  // bumped by refreshOneEntry per converged entry
```

golangci-lint (2.11.4)

[error] field convergenceCount is unused

(unused)